app/controllers/hiring_staff/vacancies_controller.rb

```diff
@@ -1,11 +1,8 @@
 class HiringStaff::VacanciesController < HiringStaff::Vacancies::ApplicationController
-  def index
-    @vacancies = school.vacancies.all
-  end
-
   def show
-    vacancy = school.vacancies.published.find(id)
+    vacancy = school.vacancies.find(id)
     @vacancy = VacancyPresenter.new(vacancy)
+    flash.now[:alert] = t('vacancies.draft') if @vacancy.status == 'draft'
```

Comment:
You could just ask the vacancy for this instead of the presented object but saying `if vacancy.draft?`